source/draw/RasterYuvPlanar.ooc

```diff
@@ -67,12 +67,8 @@ RasterYuvPlanar: abstract class extends RasterPlanar {
 	}
 	distance: func (other: Image) -> Float {
 		result := 0.0f
-		if (!other)
+		if (!other || (this size != other size) || !other instanceOf?(This))
 			result = Float maximumValue
```

Comment:
No. `RasterYuvPlanar` is an abstract class, it does not have the `convertFrom` method.